frontend/src/App.tsx

```diff
@@ -46,6 +49,10 @@ function App() {
               path="/"
               element={<NotesPage loggedInUser={loggedInUser} />}
             />
+            <Route
+              path="/trash"
+              element={<TrashPage loggedInUser={loggedInUser} />}
```

Comment:
is there a way to not repeat passing loggedInUser to every route?

Reply:
I can fetch it inside every route but it's not better